src/sentry/static/sentry/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -208,12 +249,37 @@ class AbstractIntegrationDetailedView<
     );
   }
 
-  //Returns the content shown in the top section of the integration detail
-  renderTopSection() {
+  renderAddInstallButton(configurations = true) {
```

Comment:
@NisanthanNanthakumar Two things:
1. I think `hasConfigurations` is a better variable name
2. Please add an explicit type to the input

Reply:
@NisanthanNanthakumar This is a bit confusing to me because we are saying that `configurations=true` even if there might be no configurations depending on where it is called. Probably should pick a different variable name than `hasConfigurations` or `configurations`. Maybe `hideButtonIfDisabled`?